google-http-client-apache-v2/src/main/java/com/google/api/client/http/apache/v2/ApacheHttpTransport.java

```diff
@@ -155,19 +155,14 @@ public static HttpClientBuilder newDefaultHttpClientBuilder() {
                     .setSndBufSize(8192)
                     .build();
 
-    PoolingHttpClientConnectionManager connectionManager =
-            new PoolingHttpClientConnectionManager(-1, TimeUnit.MILLISECONDS);
-    // Disable the stale connection check (previously configured in the HttpConnectionParams
-    connectionManager.setValidateAfterInactivity(-1);
-
     return HttpClientBuilder.create()
             .useSystemProperties()
             .setSSLSocketFactory(SSLConnectionSocketFactory.getSocketFactory())
```

Comment:
Would you remove this? By default, it will create and use `SSLConnectionSocketFactory.getSocketFactory()`. If this is set, I can't set `.setSSLContext()` or `.setSSLHostnameVerifier()` as they are overridden in the same manner with `setConnectionManager()`.

Reply:
Hmm... that may not exactly be same when `useSystemProperties()` is set.

Nonetheless, I think I'm OK either way, since I can explicitly unset the factory by `.setSSLSocketFactory(null)`, so not a big deal.